gslb/ingestion/gdp_controller.go

```diff
@@ -317,6 +317,15 @@ func GDPSanityChecks(gdp *gdpalphav2.GlobalDeploymentPolicy) error {
 		}
 	}
 
+	// Health monotor validity
+	if len(gdp.Spec.HealthMonitorRefs) != 0 {
+		for _, hmRef := range gdp.Spec.HealthMonitorRefs {
+			if !isHealthMonitorRefValid(hmRef) {
+				return fmt.Errorf("health monitor ref %s is invalid", hmRef)
```

Comment:
what if the other refs are valid?

Reply:
we would still return, because we have an invalid ref here @sudswasavi

Reply:
would you reflect this information on the status of the GDP?